content/en/docs/zero-code/java/agent/instrumentation/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Instrumentation configuration
+linkTitle: Instrumentation config
+weight: 100
+---
+
+Common settings that apply to multiple instrumentations at once.
+
+### Peer service name
+
+The
+[peer service name](/docs/specs/semconv/general/attributes/#general-remote-service-attributes)
+is the name of a remote service to which a connection is made. It corresponds to
+`service.name` in the [resource](/docs/specs/semconv/resource/#service) for the
+local service.
+
+{{% config_option name="otel.instrumentation.common.peer-service-mapping" %}}
+
+Used to specify a mapping from host names or IP addresses to peer services, as a
+comma-separated list of `<host_or_ip>=<user_assigned_name>` pairs. The peer
+service is added as an attribute to a span whose host or IP address match the
+mapping.
+
+For example, if set to the following:
+
+    1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api
```

Comment:
I'd favor fenced code blocks rather than indentation, but in the end it's your call. If you fix it here, fix it further below too.

~~~suggestion
```text
1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api
```
~~~